build.gradle

```diff
@@ -98,6 +41,9 @@ android {
             testCoverageEnabled = true
         }
     }
+    dexOptions {
+        preDexLibraries false
```

Comment:
Why did this need to change? And why is it false vs Timeclock: `preDexLibraries PREDEX.toBoolean()`?

Reply:
Talked about via skype